README.md

```diff
@@ -43,6 +43,32 @@ Make sure you have the following tools from the
  - `ipmi-dcmi`
  - `bmc-info`
 
+
+### How to run as non root
+
+When running this exporter as non root, you should the followings.
+
+   1. add sudoers files to permit the above commands
```

Comment:
maybe better "permit the _following_ commands"? The commands are not above that line.